go.mod

```diff
@@ -1,34 +1,10 @@
 module github.com/fullstorydev/hauser
 
 require (
-	cloud.google.com/go v0.34.0
+	cloud.google.com/go v0.36.0
 	github.com/BurntSushi/toml v0.3.1
-	github.com/aws/aws-sdk-go v1.16.6
+	github.com/aws/aws-sdk-go v1.17.12
 	github.com/lib/pq v1.0.0
-	github.com/nishanths/fullstory v0.0.0-20180815131316-5c231e3c1db8 // this will need to be updated
-	github.com/pkg/errors v0.8.0
-	golang.org/x/sys v0.0.0-20181029174526-d69651ed3497
```

Comment:
Did you mean to remove this line? I see this package referenced in the `go.sum` file, so it's a little strange that it's no longer relevant here.

Reply:
No I did not add it manually. I'm now curious what causes deps to pop in here or not. Maybe it was added before due to one of the `replacement` modules referring to the package. Now that we're not using replacements, perhaps it just uses whatever version of the package is specified by that other module's `go.mod`...